storybook/cypress/e2e/viewer-grid.cy.ts

```diff
@@ -59,7 +59,7 @@ describe("Viewer Grid", () => {
     cy.get("dso-viewer-grid").shadow().find(".overlay-close-button").should("be.focused");
   });
 
-  it.skip("should emit closeOverlay on escape", () => {
+  it("should emit closeOverlay on escape", () => {
```

Comment:
@LukeVerlaan hier stond nog een `.skip`, tegelijkertijd faalt deze test regelmatig in CI. Kan jij deze nog oplossen?

Die `.wait(100)` kunnen nog wel eens de oorzaak zijn. Een absolute test kan de oplossing zijn.

Ik kan het lokaal reproduceren met `yarn e2e`.

Reply:
Fixed

Reply:
🎉